server/middlewares/validators/videos/videos.ts

```diff
@@ -534,3 +646,11 @@ async function isVideoAccepted (req: express.Request, res: express.Response, vid
 
   return true
 }
+
+async function addDurationToVideo (videoFile: any) {
+  const duration: number = await getDurationFromVideoFile(videoFile.path)
+
+  if (isNaN(duration)) throw new Error("Couldn't get video duration")
```

Comment:
Is there a purpose that the `typescript-eslint/quotes` rule is set to `off`?

Reply:
I think I disabled this rule because it was too strict: we want to use single quotes first but allow backticks when the string contains single quotes (to avoid escaping)